components/molecules/AlgoliaResults/Templates/SearchResults.js

```diff
@@ -0,0 +1,73 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import {connectStateResults, InfiniteHits} from 'react-instantsearch-dom'
+import styles from '../AlgoliaResults.module.css'
+import Hit from './Hit'
+import NoResults from './NoResults'
+import PostType from '../Facets/PostType'
+import Authors from '../Facets/Authors'
+import CustomClearRefinements from '../Refinements/CustomClearRefinements'
+import Sort from '../Facets/Sort'
+
+/**
+ * Refinement config passed into Algolia facets.
+ */
+const refinements = {
+  limit: 3,
+  translations: {
+    showMore(expanded) {
+      return expanded ? 'Less' : 'More'
+    }
+  }
+}
+
+/**
+ * Component for rendering search results.
+ */
+const SearchResults = connectStateResults(({searchResults, indexName}) => {
```

Comment:
@dcooney Can this be a function declaration instead of expression?

Reply:
@gregrickaby I wasn't sure on the syntax of how to do this.

Reply:
It's fine. Thank you. 👍🏻